src/newSeed/stage4.ts

```diff
@@ -86,19 +87,20 @@ export class Stage4 extends BaseStage {
     if (this.endTime) {
       endTimes = this.endTime.split(":");
     }
-
     if (!this.seedConfig.seedDetails.pricePerToken || this.seedConfig.seedDetails.pricePerToken === "0") {
       message = "Please enter a value for Funding Tokens per Seed Token";
     } else if (!this.seedConfig.seedDetails.fundingTarget || this.seedConfig.seedDetails.fundingTarget === "0") {
       message = "Please enter a non-zero value for the Funding Target";
     } else if (!this.seedConfig.seedDetails.fundingMax || this.seedConfig.seedDetails.fundingMax === "0") {
       message = "Please enter a non-zero number for the Funding Max";
+    } else if (BigNumber.from(this.seedConfig.seedDetails.fundingTarget).gt(this.seedConfig.seedDetails.fundingMax)) {
+      message = "Please enter a value for Funding Target smaller than Funding Max";
+    } else if (this.seedConfig.tokenDetails.maxSeedSupply && Number(fromWei(this.seedConfig.seedDetails.fundingMax)) > Number(fromWei(this.seedConfig.tokenDetails.maxSeedSupply)) * Number(fromWei(this.seedConfig.seedDetails.pricePerToken))) {
```

Comment:
Our practice everywhere is to convert the bignumber to a string and pass that to `numberService.fromString`.  Can you please follow that here?  Thanks.